src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/SupportController.java

```diff
@@ -118,13 +118,13 @@ private void resendStaffNotificationsOnClaimIssued(final Claim claim, final Stri
             final String fullName = userService.getUserDetails(authorisation).getFullName();
 
             claimIssuedStaffNotificationHandler
-                .onClaimIssued(new ClaimIssuedEvent(claim, pinResponse.getPin(), fullName));
+                .onClaimIssued(new ClaimIssuedEvent(claim, pinResponse.getPin(), fullName, authorisation));
```

Comment:
can you pass a User down not the authorisation header?

Reply:
The enclosing method has been passed with authorisation and not the User.